stack-9.8.1.yaml

```diff
@@ -0,0 +1,14 @@
+resolver: nightly-2023-10-17
+compiler: ghc-9.8.1
+compiler-check: match-exact
```

Comment:
How is that exact match working?  Shouldn't this trigger an error in the light of: https://github.com/commercialhaskell/stackage-snapshots/blob/bcb374ae9c949ec1332d3635bdf755e538d5cea4/nightly/2023/10/17.yaml#L13012

Reply:
There is this issue "closed as completed":
- https://github.com/commercialhaskell/stack/issues/4282

Reply:
Ah, I think that `compiler:` was meant to match the compiler of the snapshot, but it seem to be the compiler used to build the present package.  I have always misunderstood this, it seems.